docs/src/components/home/DesignSystemComponents.tsx

```diff
@@ -0,0 +1,268 @@
+      },
+    },
+    [theme.breakpoints.up('md')]: {
+      marginTop: -1,
+      '&:not(:nth-of-type(3n + 1))': {
+        marginLeft: -1,
+      },
+      '&:first-of-type': {
+        borderTopLeftRadius: theme.shape.borderRadius,
+      },
+      '&:last-of-type': {
+        borderBottomRightRadius: theme.shape.borderRadius,
+        borderStyle: 'dashed',
+      },
+      '&:nth-last-of-type(3)': {
+        borderBottomLeftRadius: theme.shape.borderRadius,
+      },
+      '&:nth-of-type(3)': {
+        borderTopRightRadius: theme.shape.borderRadius,
+      },
+    },
+  },
+}));
+
+function Demo({
+  name,
+  children,
+  control,
+}: {
+  name: string;
+  children: React.ReactElement;
+  control?: { prop: string; values: Array<string> };
+}) {
+  const [propValue, setPropValue] = React.useState(control ? control.values[0] : '');
+  return (
+    <Box sx={{ display: 'flex', flexDirection: 'column', height: '100%' }}>
+      {control ? (
+        <Box sx={{ minHeight: 40, ml: -1, mt: -1 }}>
+          <Tabs
+            value={propValue}
+            onChange={(event, value) => setPropValue(value)}
+            sx={{
+              minHeight: 'initial',
+              '& .MuiTabs-indicator': {
+                bgcolor: 'transparent',
+                '&:before': {
+                  height: '100%',
+                  content: '""',
+                  display: 'block',
+                  width: (theme) => `calc(100% - ${theme.spacing(2)})`,
+                  bgcolor: 'primary.main',
+                  position: 'absolute',
+                  top: 0,
+                  left: (theme) => theme.spacing(1),
+                },
+              },
+              '& .MuiTab-root': {
+                px: 1,
+                pt: 0.5,
+                minWidth: 'initial',
+                minHeight: 'initial',
+                fontWeight: 'medium',
+              },
+            }}
+          >
+            {control.values.map((value) => (
+              <Tab value={value} label={value} />
+            ))}
+          </Tabs>
+        </Box>
+      ) : null}
+      <Box sx={{ flexGrow: 1, display: 'flex', alignItems: 'center', justifyContent: 'center' }}>
+        {React.cloneElement(children, {
+          ...(control && {
+            [control.prop]: propValue,
+          }),
+        })}
+      </Box>
+      <Typography fontWeight="bold">{name}</Typography>
+    </Box>
+  );
+}
+
+const DesignSystemComponents = () => {
+  const [anchor, setAnchor] = React.useState<HTMLElement | null>(null);
+  return (
+    <Container sx={{ py: 8 }}>
+      <Typography variant="body2" color="primary" fontWeight="bold" sx={{ mb: 1 }}>
+        What do you get?
+      </Typography>
+      <Typography variant="h2" sx={{ mb: 4, maxWidth: 500 }}>
+        Simple, accessible, declaritive <GradientText>components</GradientText>.
+      </Typography>
+      <Grid>
+        <div>
+          <Demo name="Button" control={{ prop: 'size', values: ['small', 'medium', 'large'] }}>
```

Comment:
Reminder for us to use the contained variant of the Button. We're using the `text` one.